gptel.el

```diff
@@ -522,6 +522,9 @@ BUFFER is the interaction buffer for ChatGPT."
     ('org-mode (gptel--convert-markdown->org content))
     (_ content)))
 
+(defcustom gptel-openai-endpoint "https://api.openai.com/v1"
```

Comment:
This definition needs a `:type 'string` annotation. While it's not required by Emacs, please also add a `:group 'gptel` annotation for consistency with the other definitions in this package.